htgr/treat_leu/treat_leu/tests

```diff
@@ -0,0 +1,50 @@
+[Tests]
+    # Syntax tests for each of the input files
+    [init_refcube_griffin_syntax]
+        type = RunApp
+        input = 'init_refcube.i'
+        check_input = True
+        executable_pattern = 'griffin*'
+    []
+    [adj_refcube_griffin_syntax]
+        type = RunApp
+        input = 'adj_refcube.i'
+        check_input = True
+        executable_pattern = 'griffin*'
+    []
+    [ht_20r_leu_fl_griffin_syntax]
+        type = RunApp
+        input = 'ht_20r_leu_fl.i'
+        check_input = True
+        executable_pattern = 'griffin*'
+    []
+    [refcube_griffin_syntax]
+        type = RunApp
+        input = 'refcube.i'
+        check_input = True
+        executable_pattern = 'griffin*'
+    []
+
+    # Regression tests for each of the input files
+    [init_refcube_regression]
+        type = CSVDiff
+        input = 'init_refcube.i'
+        csvdiff = 'out~init_refcube.csv'
+        executable_pattern = 'griffin*'
+        prereq = 'init_refcube_griffin_syntax'
+    []
+    [adj_refcube_regression]
+        type = CSVDiff
+        input = 'adj_refcube.i'
+        csvdiff = 'adj_refcube_out.csv'
+        executable_pattern = 'griffin*'
+        prereq = 'adj_refcube_griffin_syntax'
+    []
+    [ht_20r_leu_fl_regression]
+        type = CSVDiff
+        input = 'ht_20r_leu_fl.i'
+        csvdiff = 'ht_20r_leu_fl_out.csv'
```

Comment:
Seems like a waste to actually run this file to 10 seconds. You could use the command line arguments option to change the end time to 0.5 seconds of something less. This would still allow you to check the file, but it would be faster.

In looking at the gold file, it looks like the solution doesn't really change after the first iteration. This is due to no information or feedback coming into the simulation when it is run without MultiApp, which is fine for checking. Computation time could be saved when not having to check against all 2000 time steps.

Just something to consider.